yellowbrick/classifier/rocauc.py

```diff
@@ -168,8 +169,30 @@ def score(self, X, y=None, **kwargs):
         # Compute the predictions for the test data
         y_pred = self._get_y_scores(X)
 
-        # # Classes may be label encoded so only use what's in y to compute.
-        # # The self.classes_ attribute will be used as names for labels.
+        # Check to see if shape of predictions signals a binary decision
+        if len(y_pred.shape) == 1:
```

Comment:
I was a bit confused about this for a while, I didn't understand how the dimensionality of `y_pred` determined if the decision was binary or not. I had to realize that `_get_y_scores` is calling either the `decision_function` or `predict_proba`, which should return a 2D array. 

Would it be possible to explain this a bit more in the comment above for future us who are also likely to forget as I did? 

Also, note that `len(y_pred.shape) == 1` is equivalent to `y_pred.ndim == 1`, which may also help explain what we're checking in this condition.

Reply:
Yep, it took a bit of work to figure this out for me, too -- I just added a (maybe too lengthy) description of what's happening here in a comment inside the code. Hopefully that helps others and future us! Also switched to `ndim`.

Reply:
Not too lengthy, I think it was very descriptive, thank you!